tests/break-calls/react.js

```diff
@@ -0,0 +1,12 @@
+function helloWorld() {
+  useEffect(() => {
```

Comment:
Could we also add a test for `React.useEffect(() => {...}, [...])`? (`React.useEffect` instead of `useEffect`)